libs/common-services/src/test/java/uk/gov/di/ipv/core/library/fixtures/VcFixtures.java

```diff
@@ -0,0 +1,890 @@
+                    TestVc.TestEvidence.builder()
+                            .strengthScore(4)
+                            .validityScore(2)
+                            .ci(List.of("DO2"))
+                            .ciReasons(List.of(Map.of("ci", "D02", "reason", "NoMatchingRecord")))
+                            .build());
+
+    List<TestVc.TestEvidence> FRAUD_EVIDENCE =
+            List.of(
+                    TestVc.TestEvidence.builder()
+                            .checkDetails(null)
+                            .txn("RB000103490087")
+                            .identityFraudScore(1)
+                            .build());
+
+    List<TestVc.TestEvidence> FRAUD_FAILED_EVIDENCE =
+            List.of(
+                    TestVc.TestEvidence.builder()
+                            .checkDetails(null)
+                            .txn("RB000103490087")
+                            .identityFraudScore(0)
+                            .build());
+
+    List<TestVc.TestEvidence> FRAUD_EVIDENCE_SCORE_ONE =
+            List.of(
+                    TestVc.TestEvidence.builder()
+                            .txn("RB000180729610")
+                            .identityFraudScore(2)
+                            .checkDetails(
+                                    List.of(
+                                            Map.of(
+                                                    "checkMethod", "data",
+                                                    "fraudCheck", "mortality_check"),
+                                            Map.of(
+                                                    "checkMethod", "data",
+                                                    "fraudCheck", "identity_theft_check"),
+                                            Map.of(
+                                                    "checkMethod", "data",
+                                                    "fraudCheck", "synthetic_identity_check"),
+                                            Map.of(
+                                                    "txn", "RB000180729626",
+                                                    "checkMethod", "data",
+                                                    "fraudCheck", "impersonation_risk_check"),
+                                            Map.of(
+                                                    "checkMethod", "data",
+                                                    "activityFrom", "1963-01-01",
+                                                    "identityCheckPolicy", "none")))
+                            .build());
+
+    List<TestVc.TestEvidence> FRAUD_EVIDENCE_CRI_STUB_CHECK =
+            List.of(
+                    TestVc.TestEvidence.builder()
+                            .checkDetails(null)
+                            .type(CRI_STUB_CHECK_EVIDENCE_TYPE)
+                            .txn("some-uuid")
+                            .identityFraudScore(1)
+                            .build());
```

Comment:
Is this needed as well as the other fraud score 1 evidence block above, or can we reduce to just using one of them?